src/clients/storage/StorageClient.cpp

```diff
@@ -218,11 +218,12 @@ StorageRpcRespFuture<cpp2::GetPropResponse> StorageClient::getProps(
     req.set_space_id(param.space);
     req.set_parts(std::move(c.second));
     req.set_dedup(dedup);
-    if (vertexProps != nullptr) {
```

Comment:
Why change this?

Reply:
wait fix in https://github.com/vesoft-inc/nebula/pull/3443

Reply:
The same. Fixing something else?

Reply:
@cangfengzhs Please check why vertexProp equals to null, maybe not caused by #3443

Reply:
`FetchVerticesPlanner::buildVertexProps` return a `nullptr` when props is empty